libs/designsystem/src/lib/components/icon/icon.component.ts

```diff
@@ -1,9 +1,15 @@
 import { Component, HostBinding, Input, OnChanges, SimpleChanges } from '@angular/core';
 
-import { kirbyIconSettings } from './kirby-icon-settings';
-import { Icon } from './icon-settings';
 import { IconRegistryService } from './icon-registry.service';
+import { Icon } from './icon-settings';
+import { kirbyIconSettings } from './kirby-icon-settings';
 
+export enum IconSizes {
```

Comment:
We generally use singular form for enums:
```suggestion
export enum IconSize {
```